<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clone_dialog_net</class>
 <widget class="QWidget" name="clone_dialog_net">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string extracomment="MBR sichern/zurückschreiben">Clone hard drive</string>
  </property>
  <widget class="QPushButton" name="bt_end">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>640</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string extracomment="Abbrechen">Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_save">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>640</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string extracomment="MBR sichern/zurückschreiben">Save / restore MBR</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_hd">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>191</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string extracomment="MBR Sicherungsverzeichnis:">Exist harddrive:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rdbt_image_save">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>261</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string extracomment="sdb">Image einer Festplatte erstellen:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rdbt_image_restore">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>301</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string extracomment="sdc">Image einer Festplatte zurückschreiben:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_exist">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>181</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>471</width>
     <height>161</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>54</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>seconds</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SekundeElapsed">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>0</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>54</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>minutes</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="MinuteElapsed">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>91</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>elapsed time:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>54</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>seconds</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SekundeRemaining">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>54</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>minutes</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="MinuteRemaining">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>105</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>remaining time:</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>421</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_break">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>120</y>
      <width>24</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>24</width>
      <height>21</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>24</width>
      <height>21</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>24</width>
      <height>21</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>21</width>
      <height>21</height>
     </size>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string extracomment="Break process"/>
    </property>
    <property name="icon">
     <iconset resource="../icon.qrc">
      <normaloff>:/src/images/cancel.png</normaloff>:/src/images/cancel.png</iconset>
    </property>
    <property name="shortcut">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="savedBytes">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>already saved:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_hd_size">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>90</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>MB</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>hours</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="HourElapsed">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>hours</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="HourRemaining">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_folder">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>640</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>f</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_free_folder">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>481</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Image">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>171</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string extracomment="Name der Sicherungsdatei:">Image name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Dateiname">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>201</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_rechner_net">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>360</y>
     <width>251</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>330</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string extracomment="Netzrechnername:">Network computer  name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Exist Image:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_folder_free">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>400</y>
     <width>16</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>l</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="restore_file_name_txt">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>410</y>
     <width>360</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <zorder>pushButton_folder</zorder>
  <zorder>bt_end</zorder>
  <zorder>button_save</zorder>
  <zorder>lbl_hd</zorder>
  <zorder>rdbt_image_save</zorder>
  <zorder>rdbt_image_restore</zorder>
  <zorder>listWidget_exist</zorder>
  <zorder>frame_2</zorder>
  <zorder>listWidget_free_folder</zorder>
  <zorder>lbl_Image</zorder>
  <zorder>lineEdit_Dateiname</zorder>
  <zorder>txt_rechner_net</zorder>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
  <zorder>pushButton_folder_free</zorder>
  <zorder>restore_file_name_txt</zorder>
 </widget>
 <resources>
  <include location="../icon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>listWidget_free_folder</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>pushButton_folder_free</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>74</x>
     <y>412</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
